docs/reference/js/api.md

```diff
@@ -5,38 +5,129 @@ description: Developer resources for working in JavaScript with IPFS, the InterP
 
 # IPFS in JavaScript
 
-::: callout
-[Explore js-ipfs through interactive coding challenges at ProtoSchool](https://proto.school/course/ipfs)
+## JavaScript libraries
+
+There are two main JavaScript libraries for working with IPFS. Both work in Node.js and modern web browsers.
```

Comment:
I'd suggest adding the names of the two libraries here.

Reply:
added